docs/site/translation.md

```diff
@@ -301,6 +285,23 @@ See [Recent changes](https://cldr.unicode.org/translation#recent-changes) for ad
 |  🕳️ | Missing | Completely missing |
 | ⬆️ | Inherited | Used in combination with ✖️ and ❌ |
 
+### Enhanced "Show Hidden"
+🆕 If a field contains characters that are invisible or certain characters that look like others, a special Show Hidden bar will appear below the field that helps distinguish them.
+For example, see [Example Hidden] — here is a screen-shot.
+
```

Comment:
Do we have a part of the info site that explains how to insert these? If so, should we link to it. If not, should we create one for vetters of languages who may not have thought about this before?

Reply:
There is a ticket for allowing input of the escaped forms generally (we allow them now just for exemplars). https://unicode-org.atlassian.net/browse/CLDR-18457
So we could broaden that out to some or all paths.